VMT/templates/KCSA.md

```diff
@@ -0,0 +1,71 @@
+# Kata Containers security advisories (KCSA) template
+
+> **FIXME** This template most likely needs review and rewrite.
+
+The document is first submitted as a markdown formatted description to the KCSA project using this template:
```

Comment:
It sort of is, as are the other templates in this template dir. The idea being you can copy, edit, and post pretty quickly by filling it out. Sure, there is a little bit of explanatory text in some of them to describe the process. If we did lift that text out then I think it would be up into the main `VMT.md` doc in the dir above. If we think we'd like the templates to be a pure copy/edit/paste `md` each, then I can do that split-n-lift.